src/modules/xplorers-ai/xplorersAIChatBox.tsx

```
@@ -0,0 +1,42 @@
+      aria-live="polite"
+      className="chat-content flex max-h-[58vh] flex-col space-y-4 overflow-auto rounded-xl bg-white bg-opacity-50 p-3 shadow-md"
+    >
+      <div className="flex flex-col space-y-4">
+        {messages?.map((message: ChatMessage, index: number) => (
+          <div key={index} className="">
+            {message?.content && (
+              <div
+                className={`rounded p-2 ${message.role === "user" ? "bg-blue-200" : "bg-white"}`}
+              >
+                <span
+                  dangerouslySetInnerHTML={{ __html: marked(message.content) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)